opentracing-example/src/main/scala/zio/telemetry/opentracing/example/http/StatusesService.scala

```diff
@@ -1,46 +1,37 @@
 package zio.telemetry.opentracing.example.http
 
-import io.circe.Encoder
 import io.opentracing.propagation.Format.Builtin.{ HTTP_HEADERS => HttpHeadersFormat }
 import io.opentracing.propagation.TextMapAdapter
 import io.opentracing.tag.Tags
-import org.http4s.circe.jsonEncoderOf
-import org.http4s.dsl.Http4sDsl
-import org.http4s.{ EntityEncoder, HttpRoutes }
 import sttp.model.Uri
 import zio.clock.Clock
-import zio.interop.catz._
 import zio.telemetry.opentracing.OpenTracing
-import zio.UIO
-import zio.ZLayer
+import zio.{ UIO, ZIO, ZLayer }
+import zhttp.http.HttpApp
+import zhttp.http._
+import zio.json._
 
 import scala.collection.mutable
 import scala.jdk.CollectionConverters._
 
 object StatusesService {
-
-  def statuses(backendUri: Uri, service: ZLayer[Clock, Throwable, Clock with OpenTracing]): HttpRoutes[AppTask] = {
-    val dsl: Http4sDsl[AppTask] = Http4sDsl[AppTask]
-    import dsl._
-
-    implicit def encoder[A: Encoder]: EntityEncoder[AppTask, A] = jsonEncoderOf[AppTask, A]
-
-    HttpRoutes.of[AppTask] { case GET -> Root / "statuses" =>
+  def statuses(backendUri: Uri, service: ZLayer[Clock, Throwable, Clock with OpenTracing]): HttpApp[Clock, Throwable] = {
+    HttpApp.collectM { case Method.GET -> Root / "statuses" =>
       val zio =
         for {
           _       <- OpenTracing.tag(Tags.SPAN_KIND.getKey, Tags.SPAN_KIND_CLIENT)
-          _       <- OpenTracing.tag(Tags.HTTP_METHOD.getKey, GET.name)
+          _       <- OpenTracing.tag(Tags.HTTP_METHOD.getKey, "GET")
```

Comment:
Is there any constant for this?

Reply:
There's no need for this change.

Reply:
`GET` was defined in http4s, and the `GET` object in zio-http doesn't seem to have a `name` member. I could either use the `toString` method, or `asJHttpMethod.name`, both of which return the name of the HTTP method.

Reply:
You may use the one from sttp then.